app/store/createRootReducer.ts

```diff
@@ -97,6 +99,8 @@ const createRootReducer = () => {
     followersCompany,
     followersUser,
     followersEvent,
+    forums,
+    threads,
```

Comment:
Not to be a nerd, but please put these alphabetically in the list - not just randomly at the bottom